inference-engine/tests/functional/plugin/shared/src/behavior/layout.cpp

```diff
@@ -84,6 +84,9 @@ inline bool checkLayout(InferenceEngine::Layout layout, std::vector<size_t> &inp
 TEST_P(LayoutTest, NetWithLayout) {
     // Skip test according to plugin specific disabledTestPatterns() (if any)
     SKIP_IF_CURRENT_TEST_IS_DISABLED();
+    if (!PluginCache::get().isDeviceAvailable(targetDevice)) {
+        SKIP() << "No devices available for " << targetDevice;
+    }
```

Comment:
You can this condition to your plugin skip config file

Reply:
Not sure if I get the idea. Which file should I update in order  to do that?